src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    @extend %font-500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %font-400;
+
+    width: 100%;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    margin-bottom: 24px;
+  }
+
+  &__feedback {
+    @extend %font-400;
+
+    align-self: center;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    margin-bottom: 16px;
+
+    &-hint {
+      @extend %font-400;
+
+      font-size: 12px;
+      line-height: 18px;
+
+    }
+
+    &-value {
+      @extend%font-700;
+
+    }
```

It's not a modifiers or elements of `price`, it's a card block elements.
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

And more information about nesting in this [video](https://mate.academy/learn/css-advanced/sass#/video/925)

Delete unnecessary page breaks.

```suggestion
    &__price-hint {
      @extend %font-400;

      font-size: 12px;
      line-height: 18px;
    }

    &__price-value {
      @extend%font-700;
    }
```